cmd/clone.go

```diff
@@ -123,8 +155,8 @@ func getAllUserCloneUrls() ([]string, error) {
 }
 
 func createDirIfNotExist() {
-	if _, err := os.Stat(config.AbsolutePathToCloneTo + os.Args[1] + "_ghorg"); os.IsNotExist(err) {
-		err = os.MkdirAll(config.AbsolutePathToCloneTo, 0700)
+	if _, err := os.Stat(os.Getenv("GHORG_ABSOLUTE_PATH_TO_CLONE_TO") + args[0] + "_ghorg"); os.IsNotExist(err) {
```

Comment:
Is there any way to fully customize the name of the folder that I sync to (no ghorg suffix, name independent  from the org name)?